### PR TITLE
Fixing accessibility problems and HTML validation errors in built-in themes (Lombiq Technologies: OCORE-83)

### LINKED ISSUE #11242: HTML validation and accessibility errors in some themes
### Describe the bug
There are accessibility problems in:
- [The Agency Theme](https://github.com/OrchardCMS/OrchardCore/tree/main/src/OrchardCore.Themes/TheAgencyTheme) on the home page.
```
Asserting the accessibility analysis result failed. Check the accessibility report failure dump for details.
-------- Shouldly.ShouldAssertException : axeResult.Violations
    should be empty but had
1
    item and was
[Selenium.Axe.AxeResultItem (66331987)]

Additional Info:
Links must have discernible text: 
    <a class="btn btn-dark btn-social mx-2" href="<https://www.twitter.com/@">>
                        <i class="fab fa-twitter"></i>
                    </a>
    <a class="btn btn-dark btn-social mx-2" href="<https://www.facebook.com/@">>
                        <i class="fab fa-facebook"></i>
                    </a>
    <a class="btn btn-dark btn-social mx-2" href="<https://www.linkedin.com/in/@">>
                        <i class="fab fa-linkedin"></i>
                    </a>
    <a class="btn btn-dark btn-social mx-2" href="<https://www.twitter.com/@">>
                        <i class="fab fa-twitter"></i>
                    </a>
    <a class="btn btn-dark btn-social mx-2" href="<https://www.facebook.com/@">>
                        <i class="fab fa-facebook"></i>
                    </a>
    <a class="btn btn-dark btn-social mx-2" href="<https://www.linkedin.com/in/@">>
                        <i class="fab fa-linkedin"></i>
                    </a>
    <a class="btn btn-dark btn-social mx-2" href="<https://www.twitter.com/@">>
                        <i class="fab fa-twitter"></i>
                    </a>
    <a class="btn btn-dark btn-social mx-2" href="<https://www.facebook.com/@">>
                        <i class="fab fa-facebook"></i>
                    </a>
    <a class="btn btn-dark btn-social mx-2" href="<https://www.linkedin.com/in/@">>
                        <i class="fab fa-linkedin"></i>
                    </a>
    <a href="#">
                    <img class="img-fluid img-brand d-block mx-auto" src="/media/logos/microsoft.svg" alt="">
                </a>
    <a href="#">
                    <img class="img-fluid img-brand d-block mx-auto" src="/media/logos/google.svg" alt="">
                </a>
    <a href="#">
                    <img class="img-fluid img-brand d-block mx-auto" src="/media/logos/facebook.svg" alt="">
                </a>
    <a href="#">
                    <img class="img-fluid img-brand d-block mx-auto" src="/media/logos/ibm.svg" alt="">
                </a>
    <a class="btn btn-dark btn-social mx-2" href="#!"><i class="fab fa-twitter" aria-hidden="true"></i></a>
    <a class="btn btn-dark btn-social mx-2" href="#!"><i class="fab fa-facebook-f" aria-hidden="true"></i></a>
    <a class="btn btn-dark btn-social mx-2" href="#!"><i class="fab fa-linkedin-in" aria-hidden="true"></i></a>

```
Also there are elements with insufficient color contrasts, but I didn't include those errors. For those see: #11241

- [The Coming Soon Theme](https://github.com/OrchardCMS/OrchardCore/tree/main/src/OrchardCore.Themes/TheComingSoonTheme) on the home page.
```
Asserting the accessibility analysis result failed. Check the accessibility report failure dump for details.
-------- Shouldly.ShouldAssertException : axeResult.Violations
    should be empty but had
1
    item and was
[Selenium.Axe.AxeResultItem (14851760)]

Additional Info:
    Links must have discernible text: 
    <a class="btn btn-dark m-3" href="<https://twitter.com">>
            <i class="fab fa-twitter"></i>
        </a>
    <a class="btn btn-dark m-3" href="<https://facebook.com">>
            <i class="fab fa-facebook-f"></i>
        </a>
    <a class="btn btn-dark m-3" href="<https://instagram.com">>
            <i class="fab fa-instagram"></i>
        </a>
```
There are HTML validation errors in:
- [The Coming Soon Theme](https://github.com/OrchardCMS/OrchardCore/tree/main/src/OrchardCore.Themes/TheComingSoonTheme) on the home page.
```
  " Check the HTML validation report in the failure dump for details.
-------- Shouldly.ShouldAssertException : validationResult.Output
    should be empty but was
"12bbd89b-7645-413b-a1d3-d69909ac40b9.html
  25:55   error  The autoplay attribute is not allowed on <video>  no-autoplay
  31:320  error  Attribute "method" has invalid value "POST"       attribute-allowed-values

Ôťľ 2 problems (2 errors, 0 warnings)

More information:
  https://html-validate.org/rules/no-autoplay.html
  https://html-validate.org/rules/attribute-allowed-values.html
"
```

### To Reproduce
1. Set up Orchard Core with the [agency recipe](https://github.com/OrchardCMS/OrchardCore/blob/main/src/OrchardCore.Themes/TheAgencyTheme/Recipes/agency.recipe.json)/[coming soon recipe](https://github.com/OrchardCMS/OrchardCore/blob/main/src/OrchardCore.Themes/TheComingSoonTheme/Recipes/comingsoon.recipe.json).
2. Login and go to the homepage.
3. See that there are accessibility errors and/or HTML validation errors on the given site.

### Expected behavior
After setting up Orchard Core with the [agency recipe](https://github.com/OrchardCMS/OrchardCore/blob/main/src/OrchardCore.Themes/TheAgencyTheme/Recipes/agency.recipe.json)/[coming soon recipe](https://github.com/OrchardCMS/OrchardCore/blob/main/src/OrchardCore.Themes/TheComingSoonTheme/Recipes/comingsoon.recipe.json) on the home page, there should be no accessibility and/or HTML validation problems.

### PR DESCRIPTION
**Pull request by DemeSzabolcs**
This PR **doesn't** include fixes for: `25:55   error  The autoplay attribute is not allowed on <video>  no-autoplay` HTML validation error in [The Coming Soon Theme](https://github.com/OrchardCMS/OrchardCore/tree/main/src/OrchardCore.Themes/TheComingSoonTheme), because I think the auto-playing video is a crucial part of that theme. Also, there are overlays on the video, and it's in the background, so it's not that disturbing.

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/OrchardCMS/OrchardCore?pullRequest=11243) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/OrchardCMS/OrchardCore?pullRequest=11243) before we can accept your contribution.<br/><br/>:x: DemeSzabolcs [sign now](https://cla.dotnetfoundation.org/OrchardCMS/OrchardCore?pullRequest=11243)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/OrchardCMS/OrchardCore?pullRequest=11243) it.</sub>

**Comment by Skrypt** [comment]
Please open a PR on the appropriate repository else we will restart fixing those when we will update these themes later on.

**Comment by hishamco** [comment]
I don't think this PR should go in OC

**Comment by DemeSzabolcs** [comment]
I created PRs on the appropriate repositories:
The Coming Soon theme: https://github.com/StartBootstrap/startbootstrap-coming-soon/pull/62
The Agency theme: https://github.com/StartBootstrap/startbootstrap-agency/pull/323
The Agency theme insufficient color contrast issue: https://github.com/StartBootstrap/startbootstrap-agency/issues/322

**Comment by Piedone** [comment]
I don't think there's any point in waiting more for those theme projects that didn't react to your issues. Please wrap this up with everything that's readily usable, including updating Agency, since that was fixed (https://github.com/StartBootstrap/startbootstrap-agency/pull/323).

**Comment by DemeSzabolcs** [comment]
>  including updating Agency

I just checked, I already updated it here in the way that the PR was accepted in its own repository.

File: src/OrchardCore.Themes/TheAgencyTheme/Recipes/Snippets/footer.html [hunk 1]
```diff
@@ -10,9 +10,9 @@
                 </script>
             </div>
             <div class="col-lg-4 my-3 my-lg-0">
-                <a class="btn btn-dark btn-social mx-2" href="#!"><i class="fab fa-twitter" aria-hidden="true"></i></a>
-                <a class="btn btn-dark btn-social mx-2" href="#!"><i class="fab fa-facebook-f" aria-hidden="true"></i></a>
-                <a class="btn btn-dark btn-social mx-2" href="#!"><i class="fab fa-linkedin-in" aria-hidden="true"></i></a>
+                <a class="btn btn-dark btn-social mx-2" href="#!" aria-label="Twitter"><i class="fab fa-twitter" aria-hidden="true"></i></a>
```
Comment (Piedone) [inline src/OrchardCore.Themes/TheAgencyTheme/Recipes/Snippets/footer.html:13]: Change it to the new [X icon](https://fontawesome.com/icons/x-twitter?f=brands&s=solid) and use the label "X (Twitter)". Same for `landingpage.liquid` and the ComingSoon theme too.

File: src/OrchardCore.Themes/TheBlogTheme/Views/Content-Category.Summary.liquid [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <div class="post-preview">
     {% a display_for:Model.ContentItem %}
         <h1>
-            <i class="fas {{ Model.ContentItem.Content.Category.Icon.Text }} fa-xs" aria-hidden="true"></i>
+            <i class="{{ Model.ContentItem.Content.Category.Icon.Text }} fa-xs" aria-hidden="true"></i>
```
Comment (Piedone) [inline src/OrchardCore.Themes/TheBlogTheme/Views/Content-Category.Summary.liquid:4]: Why remove `fas`? It's used in a lot of places in OC.

Also for `Content-Category.liquid˙.

**Review by Piedone — CHANGES_REQUESTED** [review]
This does not fix https://github.com/OrchardCMS/OrchardCore/issues/11241, right?

Since due to the autoplay video remaining this does not fully fix https://github.com/OrchardCMS/OrchardCore/issues/11242 please factor that problem out to its own issue.

File: src/OrchardCore.Themes/TheBlogTheme/Views/Content-Category.Summary.liquid (hunk 2 above)
Comment (DemeSzabolcs) [reply src/OrchardCore.Themes/TheBlogTheme/Views/Content-Category.Summary.liquid:4]: https://github.com/OrchardCMS/OrchardCore/issues/11242#issuecomment-1065286678

Comment (DemeSzabolcs) [reply src/OrchardCore.Themes/TheBlogTheme/Views/Content-Category.Summary.liquid:4]: https://github.com/OrchardCMS/OrchardCore/issues/11242#issuecomment-1065301279

**Comment by DemeSzabolcs** [comment]
> This does not fix #11241, right?

No because of these:
https://github.com/StartBootstrap/startbootstrap-coming-soon/pull/62
https://github.com/StartBootstrap/startbootstrap-agency/issues/322
https://github.com/StartBootstrap/startbootstrap-bare/issues/58

> Since due to the autoplay video remaining this does not fully fix #11242 please factor that problem out to its own issue.

We talked about this internally way back. The video is in the background, so there is no way to start it manually. And the conclusion was that it could remain.

File: src/OrchardCore.Themes/TheAgencyTheme/Recipes/Snippets/footer.html (hunk 1 above)
Comment (DemeSzabolcs) [reply src/OrchardCore.Themes/TheAgencyTheme/Recipes/Snippets/footer.html:13]: Okay, but then I will need to open new PRs and wait for them to get approved too.

**Comment by Piedone** [comment]
OK!

File: src/OrchardCore.Themes/TheAgencyTheme/Recipes/Snippets/footer.html (hunk 1 above)
Comment (Piedone) [reply src/OrchardCore.Themes/TheAgencyTheme/Recipes/Snippets/footer.html:13]: This is OC code, you can just change it.

File: src/OrchardCore.Themes/TheBlogTheme/Views/Content-Category.Summary.liquid (hunk 2 above)
Comment (Piedone) [reply src/OrchardCore.Themes/TheBlogTheme/Views/Content-Category.Summary.liquid:4]: Ah, OK hen.

File: src/OrchardCore.Themes/TheComingSoonTheme/Assets/dist/index.html [hunk 3]
```diff
@@ -65,7 +65,7 @@ <h1 class="fst-italic lh-1 mb-4">Our Website is Coming Soon</h1>
         <!-- For more icon options, visit https://fontawesome.com/icons?d=gallery&p=2&s=brands-->
         <div class="social-icons">
             <div class="d-flex flex-row flex-lg-column justify-content-center align-items-center h-100 mt-3 mt-lg-0">
-                <a class="btn btn-dark m-3" href="#!" aria-label="Twitter"><i class="fab fa-twitter"></i></a>
+                <a class="btn btn-dark m-3" href="#!" aria-label="X (Twitter)"><i class="fab fa-x-twitter"></i></a>
```
Comment (Piedone) [inline src/OrchardCore.Themes/TheComingSoonTheme/Assets/dist/index.html:68]: The snippet doesn't have this change.

**Review by Piedone — CHANGES_REQUESTED** [review]

**Review by Piedone — APPROVED** [review]

**Pull request merged**